Source/KakapoServer.swift

```diff
@@ -74,19 +73,19 @@ class KakapoServer: NSURLProtocol {
         
     }
     
-    static func get(urlString: String, handler: (request: KakapoRequest) -> ()) {
+    static func get(urlString: String, handler: (request: Request) -> ()) {
         KakapoServer.routes[urlString] = (.GET, handler)
     }
     
-    static func post(urlString: String, handler: (request: KakapoRequest) -> ()) {
+    static func post(urlString: String, handler: (request: Request) -> ()) {
         KakapoServer.routes[urlString] = (.POST, handler)
     }
     
-    static func del(urlString: String, handler: (request: KakapoRequest) -> ()) {
+    static func del(urlString: String, handler: (request: Request) -> ()) {
```

Comment:
I really prefer `del` here but since the standard is `DELETE` i will go for it probably, that way it will we easier for the user